ui/appui-react/src/appui-react/widgets/WidgetDef.tsx

```diff
@@ -103,12 +106,10 @@ export class WidgetDef {
 
   public get state(): WidgetState {
     const frontstageDef = UiFramework.frontstages.activeFrontstageDef;
-    if (frontstageDef && frontstageDef.findWidgetDef(this.id)) {
-      const currentState = frontstageDef.getWidgetCurrentState(this);
-      // istanbul ignore else
-      if (undefined !== currentState) return currentState;
-    }
-    return this.defaultState;
+    const state = frontstageDef?.nineZoneState;
```

Comment:
The name `state` seems a bit generic here since the property method is already called `state` and we also pass `state` to `getWidgetState` function.

Reply:
Renamed to `nineZone`.

Reply:
I see you changed it in `getWidgetState` but it would be good to change it here as well.

Reply:
Renamed.